verilog/parser/verilog.y

```diff
@@ -2398,15 +2399,14 @@ function_declaration
   | TK_function lifetime_opt
     function_return_type_and_id ';'
     function_item_list
-    statement_or_null_list_opt
+    statement_or_null_list_opt 
```

Comment:
Trailing whitespace.
Ideally, if you have configured your editor with http://editorconfig.org/ the [.editorconfig](https://github.com/chipsalliance/verible/blob/master/.editorconfig) should make sure that this is automatically taken care of.